diff --git a/data/org.librehunt.Organizer.appdata.xml.in b/data/org.librehunt.Organizer.appdata.xml.in
index 76fa1bd..3ad5707 100644
--- a/data/org.librehunt.Organizer.appdata.xml.in
+++ b/data/org.librehunt.Organizer.appdata.xml.in
@@ -17,6 +17,7 @@
       <li>Constantly improving, just inform the developer! (in the link below)</li>
     </ul>
 	</description>
+  <url type="homepage">https://gitlab.gnome.org/aviwad/organizer</url>
   <url type="bugtracker">https://gitlab.gnome.org/aviwad/organizer/issues</url>
   <project_group>GNOME</project_group>
 
